static/js/publisher/release/components/releasesConfirm.js

```diff
@@ -5,64 +5,128 @@ import { connect } from "react-redux";
+            {closesCount > 0 && (
+              <Fragment>
+                <span className="p-tooltip">
+                  <span className="p-help">
+                    {closesCount} channel
+                    {closesCount > 1 ? "s" : ""}
+                  </span>
+                  <span className="p-tooltip__message" role="tooltip">
+                    Close channels: {pendingCloses.join(", ")}
+                  </span>
+                </span>{" "}
+                to close.
+              </Fragment>
+            )}{" "}
+            {releasesCount > 0 && (
+              <Fragment>
+                <span className="p-tooltip">
+                  <span className="p-help">
+                    {releasesCount} revision
+                    {releasesCount > 1 ? "s" : ""}
+                  </span>
+                  <span className="p-tooltip__message" role="tooltip">
+                    Release revisions:
+                    <br />
+                    {Object.keys(pendingReleases).map(revId => {
+                      const release = pendingReleases[revId];
 
-                    return (
-                      <span key={revId}>
-                        <b>{release.revision.revision}</b> (
-                        {release.revision.version}){" "}
-                        {release.revision.architectures.join(", ")} to{" "}
-                        {release.channels.join(", ")}
-                        {"\n"}
-                      </span>
-                    );
-                  })}
-                </span>
-              </span>{" "}
-              to release.
-            </Fragment>
-          )}{" "}
-          {closesCount > 0 && (
-            <Fragment>
-              <span className="p-tooltip">
-                <span className="p-help">
-                  {closesCount} channel
-                  {closesCount > 1 ? "s" : ""}
-                </span>
-                <span className="p-tooltip__message" role="tooltip">
-                  Close channels: {pendingCloses.join(", ")}
-                </span>
-              </span>{" "}
-              to close.
-            </Fragment>
-          )}{" "}
+                      return (
+                        <span key={revId}>
+                          <b>{release.revision.revision}</b> (
+                          {release.revision.version}){" "}
+                          {release.revision.architectures.join(", ")} to{" "}
+                          {release.channels.join(", ")}
+                          {"\n"}
+                        </span>
+                      );
+                    })}
+                  </span>
+                </span>{" "}
+                to release.
+              </Fragment>
+            )}{" "}
+          </p>
+          {releasesCount > 0 && (
+            <div className="p-releases-confirm__rollout">
+              <p className="u-no-margin--bottom">
+                Release {releasesCount} revision
```

Comment:
Ah this is ok on Firefox (aside from the space after the input):
![Screenshot_2019-09-13  Releases and revision history for lukewh-test ](https://user-images.githubusercontent.com/479384/64847505-119b0d00-d607-11e9-8286-b1a9b90ed05b.png)
Will take a look.
It's actually a "number" field... but vanilla doesn't seem to support it so maybe a dropdown would be better - there's just the question of "what should we show" for example for snap a 1% might be 100 people, but on snap b it might be 1 person. Having the lowest be 10% would mean 1000 people for snap a, which the publisher might not feel comfortable about.